fix: kpt force named "false" in schema

zevisert opened the pull request:
<!-- Thank you for your contribution! -->

<!-- Include if applicable: -->
Fixes: None
**Related**: None
**Merge before/after**: None
^ I didn't create an issue, I just edited directly on github 🥲

**Description**
<!-- Describe your changes here. The more detail, the easier the review! -->
I think the intent for the current schema at `skaffold.yaml:deploy.kpt.false` was meant to be `deploy.kpt.force`.
https://skaffold.dev/docs/references/yaml/#deploy-kpt-false

**User facing changes (remove if N/A)**
<!-- Describe any user facing changes this PR introduces. -->
Users using `deploy.kpt.false` will need to change to `deploy.kpt.force`.

<!-- "Before" and "After" sections work great - bonus points for screenshots! -->
<!-- Be sure all docs have been updated as well! -->

**Follow-up Work (remove if N/A)**
<!-- Mention any related follow up work to this PR. -->
- I don't know how skaffold fix works, this could be auto fixed.

<!--
Please be sure your PR includes unit tests - we don't merge code that brings down test coverage! 
Integration tests are sometimes an appropriate substitute. 
-->

zevisert commented:
Anything I can do to merge this? It's been sitting open for a while now. I could rebase, and I could also look into `skaffold fix`

zevisert commented:
Rebased on `main`. There is no need for any work in any `pkg/skaffold/schema/<version>/upgrade.go` for `skaffold fix` to function correctly, since this field was always parsed correctly, so the internal representation is/was correct. 

```console
$ cat << END > skaffold.yaml  
apiVersion: skaffold/v4beta1
kind: Config
metadata:
  name: PR9074
deploy:
  kpt:
    "false": true
END

$ ./out/skaffold fix
apiVersion: skaffold/v4beta8
kind: Config
metadata:
  name: PR9074
deploy:
  kpt:
    force: true
```

renzodavid9 commented:
Hey @zevisert, thanks for opening this PR. Could you please rebase it? This will add your changes on top of the new schema version created. Also, according to the failing tests, you'll need to run `make generate-schemas` and `make generate-schemas-v2`, those commands will update the files used as a source to render the docs page. Thanks! and sorry for the delay here.

zevisert commented:
Rebased, and schemas updated. Thanks for interpreting the test results for me - I ran and reverted both commands a couple of times, it seemed like both had the same effect. I committed the updated schema.

No worries about the time taken, there's lots of other open issues here!

zevisert commented:
Rebased once more to sign commits
